### ISSUE TITLE
`@stylistic/jsx-indent-props`: Broken in ternary expression

### ISSUE THREAD

**Issue by ej612** [eslint-stylistic/eslint-stylistic#439]
### Validations

- [X] Follow our [Code of Conduct](https://github.com/eslint-stylistic/eslint-stylistic/blob/main/CODE_OF_CONDUCT.md)
- [X] Read the [Contributing Guide](https://eslint.style/contribute/guide).
- [X] Check that there isn't already an issue that reports the same bug to avoid creating a duplicate.

### Describe the bug

Hi there!

This is my eslint config:
```
'@stylistic/indent': ['error', 2],
'@stylistic/jsx-indent': ['error', 2],
'@stylistic/jsx-indent-props': ['error', 2],
```

This is my code:
```tsx
const component = (true
  ? <button
    onClick={() => {}}
  />
  : null
);
```

Which shows the following error:
![image](https://github.com/eslint-stylistic/eslint-stylistic/assets/110408767/a5476ef7-e1f8-4df3-89d3-cbd4834da3a6)

I would expect that line to be indented by 4 spaces, analogous to be following code, which doesn't display an error:
```ts
const obj = (true
  ? {
    a: 1 // <- indented by 4 spaces
  }
  : null
);
```

Auto-fixing doesn't work either, because if I indent that line by 6 spaces, as expected by `jsx-indent-props`, I now get an error from `indent` (as I would expect):
![image](https://github.com/eslint-stylistic/eslint-stylistic/assets/110408767/00680ffb-64f0-4e12-b741-5750a2a58514)

I therefore suspect there's something wrong with `jsx-indent-props`.
Thanks a lot in advance!

### Reproduction

https://stackblitz.com/edit/github-dgu1fv

### Contributes

- [ ] I am willing to submit a PR to fix this issue
- [ ] I am willing to submit a PR with failing tests

**Comment by Daydreamer-riri** [comment]
‌‌‌‌Hi @ej612, this is a known issue where `jsx-indent-props`, `jsx-indent`, and the `indent` rule may conflict with each other. We discussed this in issue #378, and in issue #413, `indent` has passed all tests for `jsx-indent`. `jsx-indent-props` is also on the agenda. We will gradually deprecate `jsx-indent` and `jsx-indent-props` in the future. For now, you can disable `jsx-indent` and `jsx-indent-props` and use `indent` alone for formatting.
